libraries/chain/controller.cpp

```diff
@@ -1919,7 +1919,7 @@ struct controller_impl {
             EOS_ASSERT( bsp == head, fork_database_exception, "committed block did not become the new head in fork database");
          }
 
-         if (auto dm_logger = get_deep_mind_logger()) {
+         if (auto dm_logger = get_deep_mind_logger(false)) {
```

Comment:
We do not want to log this for dry-run transactions.

Reply:
At `commit_block`, we don't know transaction types.